tests/pipelines/vq_diffusion/test_vq_diffusion.py

```diff
@@ -173,3 +234,27 @@ def test_vq_diffusion(self):
 
         assert image.shape == (256, 256, 3)
         assert np.abs(expected_image - image).max() < 1e-2
+
+    def test_vq_diffusion_classifier_free_sampling(self):
+        expected_image = load_image(
+            "https://huggingface.co/datasets/williamberman/misc/resolve/main"
+            "/vq_diffusion/teddy_bear_pool_classifier_free_sampling.png"
```

Comment:
Should be moved to the huggingface testing dataset. FWIW you might have to also regenerate the image because I get a different image on `VQDiffusionPipelineIntegrationtests#test_vq_diffusion`.

Reply:
Very cool :-) I'll move it!